include/ginkgo/core/solver/ir.hpp

```diff
@@ -70,9 +72,12 @@ namespace solver {
  * while not converged:
  *     residual = b - A solution
  *     error = solver(A, residual)
- *     solution = solution + error
+ *     solution = solution + relaxation * error
  * ```
  *
+ * When relaxation is 1 (default) is Iterative Refiement, when the relaxation is
+ * not 1 is Richardson iteration.
```

Comment:
I would reorganize this text a bit.
```suggestion
 * With `relaxation`  equal to 1 (default), the solver is Iterative Refinement, with 
 * `relaxation` equal to a value other than `1`, the solve is a Richardson iteration,
 * with possibility for additional preconditioning.
```